lib/jackal-kitchen/tester.rb

```diff
@@ -200,6 +202,29 @@ def insert_kitchen_ssh(path)
         end
       end
 
+      def update_spec_helpers(path)
+        spec_helpers = []
+        Find.find(path) do |location|
+          spec_helpers << location if location =~ /spec_helper.rb/
+        end
+        spec_helpers.each do |spec_helper|
+          content = File.readlines(spec_helper)
+          open(spec_helper, 'a') { |f|
+            f << "RSpec.configure do |config|\n"
```

Comment:
what if the file doesn't end in a newline? Will bork up syntax methinks... 
same deal here:
https://github.com/carnivore-rb/jackal-kitchen/blob/inject-spec-helper/lib/jackal-kitchen/tester.rb#L199